README.rst

```diff
@@ -245,6 +245,56 @@ Custom storage example:
           base_url = urljoin(settings.MEDIA_URL, "django_ckeditor_5/")
 
 
+Rename the uploaded images using uuid:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can rename the images using uuid value and maintaning the extension in lower case. Example if your uploaded image original name was `my-image.JPG` now would be `acb34ded-f203-431a-9005-387cc24a892e.jpg`
+
+You need to specity the settings variable with True value
+
+  .. code-block:: python
+
+      CKEDITOR_5_UPLOAD_IMAGES_RENAME_UUID = True
+
+
+Custom upload_url example:
+^^^^^^^^^^^^^^^^^^^^^^^
+You can use a custom upload_url to satisfy your needs.
+
+You need to create a custom view class that extends UploadImageView
```

Comment:
I'm not sure I complete understand this example :) Why do you need to create a custom view to change the `upload_url` ? Isn't that what the setting is used for ? The code below doesn't touch anything related to the url afaics

Reply:
For example. I want to optimize the size of the uploaded photos and save in a model which user upload a specific photo